apps/scoutgame/app/api/session/claimable-tokens/route.ts

```diff
@@ -0,0 +1,22 @@
+import { getLastWeek } from '@packages/dates/utils';
+import { getUserFromSession } from '@packages/nextjs/session/getUserFromSession';
+import { checkIsProcessingPayouts } from '@packages/scoutgame/tokens/checkIsProcessingPayouts';
+import { getClaimableTokens } from '@packages/scoutgame/tokens/getClaimableTokens';
+import { NextResponse } from 'next/server';
+
+type ClaimableTokensResponse = {
+  tokens: number;
+  processingPayouts: boolean;
+};
+
+export async function GET() {
+  const user = await getUserFromSession();
+  if (!user) {
+    return NextResponse.json({ tokens: 0 });
+  }
+
+  const processingPayouts = await checkIsProcessingPayouts({ week: getLastWeek() });
```

Comment:
Checking this seems unecessary, aren't you looking for claimable tokens for all weeks?

I think the reason we needed processingPayouts before was because we would write pointsReceipts as we processed dev activity

Reply:
No this just checks if the airdrop was deployed by our protocol. i.e as soon as our protocol sets the weekly merkle tree root we have a weekly claim ready for the week.